src/ansys/fluent/core/session_base_meshing.py

```diff
@@ -122,12 +122,8 @@ def meshing_utilities(self):
     @property
     def workflow(self):
         """Datamodel root of workflow."""
-        if not self._old_workflow:
-            self._old_workflow = WorkflowMode.CLASSIC_MESHING_MODE.value(
-                _make_datamodel_module(self, "workflow"),
-                self.meshing,
-                self.get_fluent_version(),
-            )
+        if self._old_workflow is None:
+            self._old_workflow = _make_datamodel_module(self, "workflow")
```

Comment:
We can get rid of the classic meshing workflow altogether only, right?

Reply:
Shall we do the clean-up in a separate PR?

Reply:
This is to test it at this time. If this looks good, we should check the console regressions.

Reply:
got it. Thanks.